bin/arm64_env.sh

```diff
@@ -19,10 +19,10 @@ export exportDistro=nexus3.edgexfoundry.org:10004/docker-export-distro-go-arm64:
 export deviceVirtual=nexus3.edgexfoundry.org:10004/docker-device-virtual-go-arm64:1.1.0
 export appService=nexus3.edgexfoundry.org:10004/docker-app-service-configurable-arm64:latest
 
-export vault=nexus3.edgexfoundry.org:10004/docker-edgex-vault-arm64:0.3.0
-export vaultWorker=nexus3.edgexfoundry.org:10004/docker-edgex-vault-worker-go-arm64:0.3.0
-export kongdb=postgres:9.5
-export kong=kong:1.3.0-ubuntu
```

Comment:
note that this does actually indeed need to be version 1.3, kong 1.2 and below doesn't run on arm64, so to run this at all we need kong 1.3 on arm64. Arguably we should also upgrade to kong 1.3 on amd64 as well